lib/generators/shopify_app/rotate_shopify_token_job/templates/rotate_shopify_token.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+namespace :shopify do
+  desc "Rotate shopify tokens for all active shops"
+  task :rotate_shopify_tokens, [:refresh_token] => :environment do |_t, args|
+    all_active_shops.each do |shop|
+      Shopify::RotateShopifyTokenJob.new.perform(
+        shop_domain: shop.shopify_domain,
+        refresh_token: args[:refresh_token]
+      )
+    end
+  end
+
+  # Its implementation will depend on the app configuration. Change accordingly.
+  def all_active_shops
+    Shop.all
```

Comment:
This is a bit dangerous. Even though this is only a suggestion, I would recommend using batching (maybe https://apidock.com/rails/ActiveRecord/Batches/find_in_batches).

Reply:
Fixed it in line 5, `all_active_shops.find_each`

Reply:
Could be wrong, but I think you want just `Shop` here. 
I think if you call Shop.all.find_each, you're still loading `all` the shops

Reply:
`Shop.all` returns a relation, which is lazy, so this is fine.